sycl/doc/extensions/PlatformContext/PlatformContext.adoc

```diff
@@ -0,0 +1,44 @@
+= SYCL(TM) Proposals: Platform Default Contexts
+James Brodman <james.brodman@intel.com>
+v0.1
+:source-highlighter: pygments
+:icons: font
+:y: icon:check[role="green"]
+:n: icon:times[role="red"]
+
+== Platform Default Contexts
+
+This extension adds the notion of a default SYCL context per SYCL platform. The default context for each platform contains all devices in the platform.
+
+The platform class gains three new methods:
+
+[cols="^60a,40"]
+|===
+| Member Function | Description
+
+|
+[source,c++]
+----
+context get_default_context()
+----
+
+| Returns the current default context for this `platform`
+
```

Comment:
It would be nice if the extension is mentioned in the https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/README.md